pages/common/mmls.md

```diff
@@ -0,0 +1,20 @@
+# mmls
+
+> Display the partition layout of a volume system.
+> More information: <https://wiki.sleuthkit.org/index.php?title=Mmls>.
+
+- Display the partition table stored in an image file:
+
+`mmls {{imagefile}}`
```

Comment:
```suggestion
`mmls {{/path/to/image_file}}`
```

Reply:
Thanks, I tried to keep it short. But it's more precise with a path.